pandas/tests/series/methods/test_fillna.py

```diff
@@ -909,14 +893,14 @@ def test_bfill(self):
             [0.0, 1.0, 2.0, 3.0, 4.0], index=date_range("2020-01-01", periods=5)
         )
         ts.iloc[2] = np.nan
-        tm.assert_series_equal(ts.bfill(), ts.fillna(method="bfill"))
+        tm.assert_series_equal(ts.bfill(), ts.bfill())
```

Comment:
this assertion looks no-longer-useful

Reply:
same on L883

Reply:
Thank you for reviewing this PR. I removed these useless tests.